Tranga/Server.cs

```diff
@@ -102,6 +102,22 @@ private void HandleRequest(HttpListenerContext context)
         return ret;
     }
 
+    private Dictionary<string, string> GetRequestBody(HttpListenerRequest request)
+    {
+        if (!request.HasEntityBody)
+        {
+            Log("No request body");
+            Dictionary<string, string> emptyBody = new();
+            return emptyBody;
+        }
+        Stream body = request.InputStream;
+        Encoding encoding = request.ContentEncoding;
+        StreamReader reader = new StreamReader(body, encoding);
+        string s = reader.ReadToEnd();
+        Dictionary<string, string> requestBody = JsonConvert.DeserializeObject<Dictionary<string, string>>(s);
```

Comment:
Probably want a try-catch so we dont crash every time we get a request that has for example integers, booleans, arrays, or nested objects.

Reply:
Good call